README.md

```diff
@@ -1738,13 +1738,13 @@ A bit on how all this fits together:
 ### State taxes
 
 State tax rates and rules
-[vary significantly](http://taxfoundation.org/article/state-individual-income-tax-rates-and-brackets-2015).
+[vary significantly](https://taxfoundation.org/state-individual-income-tax-rates-brackets-2018/).
 Since federal rates are much higher than state rates, you usually think of federal tax
 planning first.
 But you should also know a bit about tax rates in your state.
 
 State long-term capital gains rates
-[range widely](http://www.fool.com/personal-finance/taxes/2014/10/04/the-states-with-the-highest-capital-gains-tax-rate.aspx).
+[range widely](http://www.fool.com/personal-finance/taxes/2014/10/04/the-states-with-the-highest-capital-gains-tax-rate.aspxe).
```

Comment:
This link broke. I'll fix. (Recommend always looking at the PR after it's created to see these things.)